src/Tickets/Emails/Settings.php

```diff
@@ -0,0 +1,283 @@
+				'validation_type'     => 'email',
+			],
+		];
+
+		$new_fields = apply_filters( 'tec_tickets_emails_settings_sender_info_fields', $new_fields );
+
+		return array_merge( $fields, $new_fields );
+	}
+
+	/**
+	 * Adds Sender Info fields to Tickets Emails settings.
+	 *
+	 * @param  [] $fields Current array of Tickets Emails settings fields.
+	 * 
+	 * @return [] $fields Filtered array of Tickets Emails settings fields.
+	 */
+	public function email_styling_fields( $fields ) {
+
+		$new_fields = [
+			[
+				'type' => 'html',
+				'html' => '<h3>' . esc_html__( 'Email Styling', 'event-tickets' ) . '</h3>',
+			],
+			[
+				'type' => 'html',
+				'html' => '<p>' . esc_html__( 'Add a logo and customize link colors and footer information to personalize your communications.  If you\'d like more granular control over email styling, you can override the email templates in your theme.  Learn More', 'event-tickets' ) . '</p>',
+			],
+			static::$option_header_image_url  => [
+				'type'                => 'image',
+				'label'               => esc_html__( 'Header Image', 'event-tickets' ),
+				'size'                => 'medium',
+				'default'             => '',
+				'validation_callback' => 'is_string',
+				'validation_type'     => 'url',
+			],
+			static::$option_header_image_alignment  => [
+				'type'            => 'dropdown',
+				'label'           => esc_html__( 'Image Alignment', 'event-tickets' ),
+				'default'         => 'left',
+				'validation_type' => 'options',
+				'options'         => [
+					'left'   => esc_html__( 'Left', 'event-tickets' ),
+					'center' => esc_html__( 'Center', 'event-tickets' ),
+					'right'  => esc_html__( 'Right', 'event-tickets' ),
+				],
+			],
+			static::$option_header_bg_color  => [
+				'type'                => 'color',
+				'label'               => esc_html__( 'Header/Footer Background', 'event-tickets' ),
+				'size'                => 'medium',
+				'default'             => '#ffffff',
+				'validation_callback' => 'is_string',
+				'validation_type'     => 'color',
+			],
+			static::$option_ticket_bg_color  => [
+				'type'                => 'color',
+				'label'               => esc_html__( 'Ticket Color', 'event-tickets' ),
+				'size'                => 'medium',
+				'default'             => '#ffffff',
+				'validation_callback' => 'is_string',
+				'validation_type'     => 'color',
+			],
+			static::$option_footer_content  => [
+				'type'                => 'wysiwyg',
+				'label'               => esc_html__( 'Footer Content', 'event-tickets' ),
+				'tooltip'             => esc_html__( 'Add custom links and instructions to the bottom of your emails.', 'event-tickets' ),
+				'default'             => '',
+				'validation_type'     => 'html',
+				'settings'            => [
+					'media_buttons' => false,
+					'quicktags'     => false,
+					'editor_height' => 200,
+					'buttons'       => [
+						'bold',
+						'italic',
+						'underline',
+						'strikethrough',
```

Comment:
Can we include the alignment options here?

Reply:
@juanfra Yes! I figured we'd alter the settings for whatever we need when we get to that task and the design is finalized. For now, I was just going off the Figma design.